.github/workflows/draft-release.yml

```diff
@@ -42,6 +42,9 @@ jobs:
       with:
         go-version: 1.13
 
+    - name: Set GIT_BRANCH
+      run: echo "::set-env name=GIT_BRANCH::${GITHUB_REF#refs/tags/}"
```

Comment:
why is this `tags` instead of `heads` (as above)?

Reply:
hmm good point - now that I look at this, `GITHUB_REF` in the original one from above is wrong (and isn't even used anywhere) so I'll remove it

Reply:
Fixed